client/src/pages/spectrogram/components/annotation/__tests__/Actions.test.tsx

```diff
@@ -159,8 +162,8 @@ describe('Annotation list component', () => {
     await userEvent.click(updateButton);
 
     // Assert
-    const modal = await screen.findByLabelText('Annotation 0 Modal');
-    expect(modal).toHaveClass('modal-open');
+    const modal = await screen.queryByLabelText('Modal Dialog');
+    expect(modal).toBeInTheDocument();
```

Comment:
Is toBeVisible working here?

Reply:
It doesn't, but I have confirmed that if the modal isn't open then toBeInTheDocument fails